content/docs/apps/custom-domains.md

```diff
@@ -17,6 +17,38 @@ To make your app accessible via your custom domain name, use the [CDN service]({
 
 *Additional details are available in the [cloud.gov FedRAMP P-ATO documentation package]({{< relref "overview/security/fedramp-tracker.md#how-you-can-use-this-p-ato" >}}), including in System Security Plan controls SC-20, SC-21, SC-22, and SC-23.*
 
+### Comparison of default domains and custom domains
+
+Here's an example of the difference between a default *.app.cloud.gov domain and a custom domain. In this example, a staging application is using a default domain, and the production version of the application is using a custom domain.
```

Comment:
I find this diagram pretty confusing because you're conflating a number of different things: 

* Custom DNS/app.cloud.gov DNS 
* CDN / origin
* staging spaces / production spaces

Reply:
I think this is confusing because it's accurate 😆. Maybe we can simplify by dropping *.app.cloud.gov DNS and staging--I think the main thing we want users to understand here is how production apps work.

Reply:
I want people to understand how default domain names work as well as production custom domain names, so that they understand the difference clearly, especially because sometimes people use `*.app.cloud.gov` for their "beta" sites.

We could remove "staging" and "production" space names though, and call them something like "space A" and "space B".